### PR TITLE
Add propagation delay for letsencrypt with RFC2136

### LINKED ISSUE #1424: letsencrypt: propagation_seconds setting is not honored
Hi !

It looks like the DNS propagation seconds setting of letsencrypt is not honored :
```
[18:37:42] INFO: Selected DNS Provider: dns-ovh
[18:37:42] INFO: Use propagation seconds: 120
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator dns-ovh, Installer None
Obtaining a new certificate
Performing the following challenges:
dns-01 challenge for <REDACTED>
Waiting 30 seconds for DNS changes to propagate
Waiting for verification...
Challenge failed for domain <REDACTED>
dns-01 challenge for <REDACTED>
Cleaning up challenges
Some challenges have failed.
IMPORTANT NOTES:
 - The following errors were reported by the server:
   Domain: <REDACTED>
   Type:   dns
   Detail: DNS problem: NXDOMAIN looking up TXT for
   _acme-challenge.<REDACTED> - check that a DNS record exists
   for this domain
```

Thanks for your consideration :)

### PR DESCRIPTION
**Pull request by mateuszdrab**
Reopening a new PR to replace https://github.com/home-assistant/addons/pull/1729

Fixes #1424

### CODE REVIEW

File: letsencrypt/rootfs/etc/services.d/lets-encrypt/run [hunk 1]
```diff
@@ -96,6 +96,10 @@ elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-njalla" ]; then
     bashio::config.require 'dns.njalla_token'
     PROVIDER_ARGUMENTS+=("--authenticator" "certbot-dns-njalla:dns-njalla" "--certbot-dns-njalla:dns-njalla-credentials" /data/dnsapikey "--certbot-dns-njalla:dns-njalla-propagation-seconds" "${PROPAGATION_SECONDS}")
 
+# rfc2136
+elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "rfc2136" ]; then
```
Comment (mdegat01) [inline letsencrypt/rootfs/etc/services.d/lets-encrypt/run:100]:
```suggestion
elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-rfc2136" ]; then
```
`rfc2136` isn't an option in here:
https://github.com/home-assistant/addons/blob/d37b21354316c7266a5d7616a553bfe15da5af71/letsencrypt/config.yaml#L72-L76

Should be `dns-rfc2136`

**Review by mdegat01 — CHANGES_REQUESTED** [review]
@mateuszdrab Please update the changelog to include your addition

**Comment by mateuszdrab** [comment]
> @mateuszdrab Please update the changelog to include your addition

All done

**Review by mdegat01 — APPROVED** [review]

**Comment by mdegat01** [comment]
Thanks @mateuszdrab !

**Comment by mateuszdrab** [comment]
Oops, saw the syntax error, thanks for correcting @mdegat01

**Comment by mateuszdrab** [comment]
Oops 2, still messed up:
```
certbot [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...
Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
certificate. 
certbot: error: unrecognized arguments: --certbot-dns-rfc2136:dns-rfc2136-propagation-seconds 120
```

Will need to fix the arg to `--dns-rfc2136-propagation-seconds` as documented [here](https://certbot-dns-rfc2136.readthedocs.io/en/stable/#named-arguments)

**Comment by mateuszdrab** [comment]
https://github.com/home-assistant/addons/pull/2557

**Pull request merged**